### PR TITLE
Deeper fully connected linear network

### PR DESCRIPTION
**Pull request by blown302**
Network similar to the one running on car now. Commented out Normalization. Seemed to be pretty jumpy during training and simulation with normalization. Learning rate tuning may need to be looked into prior to using.

### CODE REVIEW

**Comment by tawnkramer** [comment]
I don't mind merging this request. But it looks very similar to the default_linear model. Should we just change the default linear model to use the two extra layers? Do you think there's value in keeping both around?

**Comment by blown302** [comment]
> I don't mind merging this request. But it looks very similar to the default_linear model. Should we just change the default linear model to use the two extra layers? Do you think there's value in keeping both around?

I didn't want to mess with the original networks, it's more of a V2 if you will. If no one but I uses it currently I can update to change the original. 

Any thoughts one how you'd like to handle newer versions?

**Comment by tawnkramer** [comment]
If it's an improvement and more of V2 then I would prefer to keep the model choices obvious and differentiate them clearly. We can modify the original model if it's a clear improvement and previous users will still be able to load older models. However, I'm testing this new deeper fc model and not noticing any improvement in the simulator. It's does achieve slightly (0.003) lower loss than the current linear model, but triples the disk size. Have you noticed any appreciable improvement over the default linear model?

**Comment by blown302** [comment]
It's definitely a larger model and seems more robust, but needs more vetting to be a replacement. The reason for adding is for the group to try new things since most of them won't jump into the code. Jack was asking for docs on what I was doing so I thought I'd throw it in to be easy as a switch rather than modifying code.

I have no scientific evidence at this point as I'm only been running it a few weeks. When I get time I'll try them on different surfaces to see what needs more transfer. The theory is it doesn't overfit as much.

On smaller datasets where the data is similar we may not see a difference. if definitley behaved differently when I tried to use different paths. Like center, left, right.

**Comment by blown302** [comment]
I'll wait until there is interest in the group/ more vetting.
